plugins/available/base.plugin.bash

```diff
@@ -4,13 +4,24 @@ about-plugin 'miscellaneous tools'
 function ips ()
 {
     about 'display all ip addresses for this host'
+    param 'optional -i'
```

Comment:
I would write something like `param '-i: (optional) display the interfaces and the corresponding ip addresses`